docs/css/custom.css~

```diff
@@ -0,0 +1,100 @@
+/* Styles by chris.tingom@scality.com Hello and thanks for all the fish! */

```

Comment:
Looks like this file was not meant to be committed

Reply:
That's an emacs dropping. 

I squashed it (and now I'm scraping it off my shoe).